src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -252,62 +255,79 @@ public async Task MutationMissingRequiredPartitionKeyValueReturnError()
         }
 
         /// <summary>
-        /// Mutation can be performed on the authorized fields because the
-        /// field `id` is an included field for the create operation on the anonymous role defined
-        /// for entity 'earth'
+        /// Create Mutation performed on the fields with different auth permissions
+        /// It throws permission denied error if role doesn't have permission to perform the operation
+        /// 'item-level-permission-role' is not applicable for create operation
         /// </summary>
         [TestMethod]
-        public async Task CanCreateItemWithAuthorizedFields()
+        [DataRow("field-mutation-with-read-permission", DataApiBuilderException.GRAPHQL_MUTATION_FIELD_AUTHZ_FAILURE, DisplayName = "When there is limited permission at field level but have full read permission")]
```

Comment:
```suggestion
        [DataRow("field-mutation-with-read-permission", DataApiBuilderException.GRAPHQL_MUTATION_FIELD_AUTHZ_FAILURE, DisplayName = "AuthZ failure for create mutation because of reference to excluded/disallowed fields.")]
```

nit: suggestion to make DisplayName more self explanatory on what's being validated.

Reply:
can extrapolate comments on this test's displaynames to the other test's display names.